README.md

```diff
@@ -1,34 +1,28 @@
 # pm2-syslog
 
-Redirect all logs of PM2 + Apps managed into `/var/log/syslog`
+Redirect all logs of PM2 + Apps managed into `syslog`
 
-## Configure OS
-
-Edit `/etc/rsyslog.conf` and uncomment:
-
-```
-# provides UDP syslog reception
-module(load="imudp")
-input(type="imudp" port="514")
-```
-
-Restart rsyslog:
-
-```
-$ sudo service rsyslog restart
-```
 
 ## Install module
 
 ```
 # Install
-$ pm2 install pm2-syslog
```

Comment:
Didn't worked for me, see https://github.com/Unitech/pm2/issues/2973 I'll have to go deeper to find out why, but I was already yacc shaving so I skipped it and only opened an issue